src/source-map.ts

```diff
@@ -18,7 +19,7 @@ export const installSourceMapSupport = () => {
 		 *
 		 * https://github.com/nodejs/node/blob/91193825551f9301b6ab52d96211b38889149892/lib/internal/errors.js#L141
 		 */
-		&& typeof Error.prepareStackTrace !== 'function'
+		&& (isFeatureSupported(prepareStackTraceWithSourcemap) || typeof Error.prepareStackTrace !== 'function')
```

Comment:
Upon reading what [the default `prepareStackTrace`](https://github.com/nodejs/node/blob/dd711d221a6cd3ffaccedf6fc810b9c2d16eedd5/lib/internal/errors.js#L95-L111) looks like, I'm okay with removing this check entirely.

The Node docs previously implied that if it was overridden sourcemaps wouldn't work expectedly, but the implementation is so simple that the default behavior can easily be replicated.

Reply:
done